### DISCUSSION TITLE
[Bug] Taskbar position is reset when opening fullscreen applications.

### DISCUSSION THREAD

**Discussion by PINPAL** [valinet/ExplorerPatcher#982] [General]
https://github.com/valinet/ExplorerPatcher/issues/14

As per this issue, functionality that maintains taskbar position on a secondary display (non-primary) was properly implemented however the taskbar will still reset to the primary display when opening fullscreen applications.

Windows Version 21H2 (OS Build 22000.527)
ExplorerPatcher 22000.469.41.16

**Observed Behavior**
When opening fullscreen applications such as games, the taskbar visually remains in place, however the "space" it occupies is moved to the primary display. This means windows on the secondary monitor will overlay the taskbar and windows on the primary monitor will not take up the full size of the display and leave a gap where the taskbar would be.

**Expected Behavior** 
Taskbar remains in position when opening fullscreen applications.

**Steps to Reproduce**
1. Ensure `Show taskbar on all displays` is disabled.
2. Move taskbar to the secondary (non-primary) monitor.
3. Launch any fullscreen application (such as games).
4. Observe taskbar visually remains on the secondary monitor but the "space" it occupies is on the primary monitor.

**Further Notes**
I have attached a short video to demonstrate the issue better:
https://user-images.githubusercontent.com/18197217/156625588-7e77779d-b3ca-4b04-8b53-b206dfc032ee.mp4

**Comment by valinet** [comment]
Yeah, I understand, but I don't know how to "fix" it. As I said multiple times, the fact that this works at all is a bug in itself. As a workaround, try to play the game in border less full screen mode.
